### PR TITLE
Feature channels

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |N|
|New feature?   |Y|	
|BC breaks?     |N|	
|Deprecations?  |N|	
|Fixed tickets  |N|

### CODE REVIEW

File: src/Controller/ChannelController.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+use EMS\CoreBundle\Twig\Table\ChannelTable;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ChannelController extends AbstractController
+{
+    /**
+     * @var LoggerInterface
+     */
```
Comment (Davidmattei) [inline src/Controller/ChannelController.php:17]: one line

**Review by AdrienHq — APPROVED** [review]

**Review by AdrienHq — APPROVED** [review]

**Review by Davidmattei — COMMENTED** [review]
👍

**Review by theus77 — COMMENTED** [review]
OK ready to me to be reviewed. It's finally done all is working I thimk

**Review by Davidmattei — APPROVED** [review]
Nice pr! Look over the code, don't see anything strange ... We should merge this as soon as possible.

**Pull request merged**